.github/workflows/gh-ci.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
         fail-fast: false
         matrix:
           os: [ubuntu-latest, ]
-          python-version: [3.7, 3.8, 3.9]
+          python-version: [3.8, 3.9]
```

Comment:
Are we still not able to run 3.10 just normally?

Reply:
Last I checked we were waiting for rdkit and openmm, I know that rdkit added py310 support in 2022.03.2 ~ 20 days ago.

I'm not sure if about openmm 7.7.0, I know it got added 2 months ago but I know they then pulled some stuff. Let's try it out!

Reply:
No good, looks like chemfiles-python is the last culprit here, I'll raise an issue upstream.

Reply:
isn't chemfiles optional? (for reasons like this...)

Reply:
Yeah this is why the py310  minimum deps tests are fine. That matrix is the "all the deps" matrix (since it's easier to run this)